lib/Nessy/Client.pm

```diff
@@ -328,7 +370,9 @@ sub _on_read_event {
 
 sub bailout {
     my($self, $message) = @_;
-    Carp::croak($message);
+    print STDERR $message,"\n";
```

Comment:
Should we prefix this, e.g. "nessy: $message"?  I noticed in the test there's just a line that says, "Connection reset by peer", which could probably use some context.